plejd/api.js

```diff
@@ -36,17 +36,12 @@ class PlejdApi extends EventEmitter {
   }
 
   updateSettings(settings) {
```

Comment:
In my installation this never seems to be called. I can see in the code it subscribes to `mqtt` topic `plejd/settings`. After a (little) bit of testing it seems like no messages are posted to any `plejd/#` channel (also tried listening to all messages but that log fills up quickly). Not important for this PR, but if anyone has any insight on when this is triggered that would be appreciated!

Reply:
That was something I started working on, to be able to reload settings without restart. Unfortunately that didn't make it into the code. So that topic is currently orphaned.

Reply:
Ok, great, thanks for that explanation. Not really relevant for this PR, but good to understand what's going on